doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -524,6 +524,36 @@ in the `library search path`_.
    \            Hello        world
    ===========  ===========  ============  ============
 
+Using a custom keyword name
+'''''''''''''''''''''''''''
+It is possible to expose a different name for a keyword instead of the
+default keyword name which maps to the method name.  This can be accomplished
+by setting the `robot_name` attribute on the method to the desired custom name.
+The decorator `robot.api.deco.robot_name` may be used as a shortcut for setting
+this attribute when used as follows:
+
+.. sourcecode:: python
+
+  from robot.api.deco import robot_name
+
+  @robot_name('Login Via User Panel')
```

Comment:
1) Old decorator name here and in the above paragraph. Should also double check is it still used somewhere else.

2) Now that the decorator is named just `keyword` and not `robot_name`, it might be a good idea to change the usage example to `@keyword(name='Login Via User Panel')`.